configs/top_config_first_run.yaml

```diff
@@ -0,0 +1,112 @@
+# Top Level Config Template for both the training and metric experiments
+
+# Config type, this can either edited or given as an argument in the gen_configs script
+config_type: both
+
+# Where to put the subconfigs
+config_dir: configs
+
+# Where (and if) to save results locally
+save_dir: results_pets
+local_save: False
+
+# Where to find the slurm template, if set to None will pick up jobscript from src/locomoset/config
+# slurm_template_path: None
+slurm_template_name: 'jobscript_template.sh'
+
+# Huggingface model names or paths to local models
+models:
+  - facebook/deit-base-distilled-patch16-224
+  - facebook/deit-tiny-patch16-224
+  - facebook/deit-small-patch16-224
+  - facebook/deit-base-patch16-224
+  - apple/mobilevit-small
+  - apple/mobilevit-xx-small
+  - google/vit-base-patch16-384
+  - google/vit-large-patch16-224
+  - google/vit-hybrid-base-bit-384
+  - microsoft/cvt-13
+  - microsoft/cvt-21-384-22k
+  - facebook/deit-base-patch16-384
+  - facebook/deit-base-distilled-patch16-384
+  - microsoft/beit-large-patch16-512
+  - microsoft/beit-base-patch16-224
+  - microsoft/beit-base-patch16-224-pt22k-ft22k
+  - microsoft/swin-large-patch4-window12-384-in22k
+  - facebook/levit-128S
+  - facebook/dinov2-small-imagenet1k-1-layer
+  - facebook/dinov2-base-imagenet1k-1-layer
+
+# HuggingFace Datasets or paths to local datasets
+dataset_names:
+  pcuenq/oxford-pets
+
+# Random seeds to use
+random_states:
+  - 42
+
+# Weights and biases
+wandb:
+  entity: edable-heath
+  project: locomoset_metrics_tests
+
+# Baskerville arguments, with separate arguments available for the different experiment types
+use_bask: True
+bask:
+  metrics:
+    job_name: 'locomoset_metric_experiment'
+    walltime: '2-0:0:0'
+    node_number: 1
+    gpu_number: 1
+    cpu_per_gpu': 36
+  train:
+    job_name: 'locomoset_train_experiment'
+    walltime: '2-0:0:0'
+    node_number: 1
+    gpu_number: 1
+    cpu_per_gpu': 36
+
+# Cache locations:
+caches:
+  datasets: ./.cache/huggingface/datasets
```

Comment:
Will these be assuming we run sbatch from the project root directory? Maybe worth documenting that/replacing with a full path to avoid accidentally creating caches in multiple places

Reply:
They now have the full path.